lib/rucio/core/rule.py

```diff
@@ -1814,7 +1810,7 @@ def get_expired_rules(total_workers, worker_number, limit=100, blocked_rules=[],
 
 
 @read_session
-def get_injected_rules(total_workers, worker_number, limit=100, blocked_rules=[], *, session: "Session"):
+def get_injected_rules(total_workers: int, worker_number: int, limit: int = 100, blocked_rules: Sequence[str] = [], *, session: "Session") -> list[tuple[str, str]]:
```

Comment:
The return type is currently `list[tuple[str]]`. but the implementation is poor. I would suggest you change it to `list[str]` and make the necessary adjustments in the implementation and in the caller.